src/main/java/com/romraider/logger/ecu/definition/EcuDataLoaderImpl.java

```diff
@@ -95,11 +95,12 @@ public void loadConfigFromXml(String loggerConfigFilePath,
         boolean valid = true;
         
         try {
-            InputStream inputStream = new BufferedInputStream(new FileInputStream(new File(loggerConfigFilePath)));
+            //InputStream inputStream = new BufferedInputStream(new FileInputStream(new File(loggerConfigFilePath)));
```

Comment:
Dead code bad. @ArtRoman if you want to replace a line it's best to remove old line and add new line, not keep both.